docs/app/Examples/elements/List/Variations/ListDividedExample.js

```diff
@@ -0,0 +1,18 @@
+import React, {Component} from 'react';
+import {List, ListItem} from 'stardust';
+
+export default class ListDividedExample extends Component {
+  render() {
+    const avatar1 = <img className='ui avatar image' src='http://lorempixel.com/100/100/people/1' />;
+    const avatar2 = <img className='ui avatar image' src='http://lorempixel.com/100/100/people/2' />;
+    const avatar3 = <img className='ui avatar image' src='http://lorempixel.com/100/100/people/3' />;
```

Comment:
We have [Faker](https://github.com/marak/Faker.js/) installed for random data. Let's import it and use it for the image `src={faker.internet.avatar()}` for consistency and brevity.  As a vain bonus, we get random avatars on every render.